TAScheduler/AcceptanceTests/admin_delete_course_test.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+from django.urls import reverse
+from TAScheduler.models import Course, Section, User, TA, InstructorToCourse
+
+class AdminDeleteCourseTestCase(TestCase):
+    def setUp(self):
+        # Create an admin user and course for testing
+        self.admin_user = User.objects.create(email_address='admin@example.com', password='password')
+        self.course = Course.objects.create(course_id=1, name='Test Course', semester='Fall', num_of_sections=1, modality='Online', credits=3)
+
+    def test_delete_course_success(self):
+
+        self.client.login(email_address=self.admin_user.email_address, password='password')
+
+        # Attempt to delete the course
+        response = self.client.post(reverse('delete_course'), {'course_id': self.course.course_id})
+        self.assertRedirects(response, reverse('course_management'))
```

Comment:
Check for a confirmation message.
Example:
.context["message"] = "successfully deleted course"